apps/sap-commerce-cloud-with-air/lambda/src/config.ts

```diff
@@ -0,0 +1,23 @@
+type EnvironmentVariable = string | null | number | undefined;
+
+function getEnvironmentVariable(
+  environmentVariableName: string,
+  fallback?: EnvironmentVariable
+): string {
+  const environmentVariableValue = process.env[environmentVariableName];
+  if (!environmentVariableValue) {
+    if (fallback) return fallback.toString();
+    throw new Error(`Missing environment variable: '${environmentVariableName}'`);
+  }
+  return environmentVariableValue;
+}
+
+export const config = {
+  stage: getEnvironmentVariable('STAGE'),
```

Comment:
what does "stage" mean in this context? (again this is curiosity question). Based on the code context, assuming this is just any environment besides production?

Reply:
Yeah there's some serverless stuff that defines the env vars of what stage we are in (and all of these env vars really) and then in the application code we can flexibly write if-else logic based on the stages we are in. Not a thing that will be necessarily used, but more of a future-proof type of useful to have type of deal